test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -100,14 +100,18 @@ describe('Test gl plot interactions', function() {
                 delay(done);
             });
 
-            it('should have', function() {
-                node = d3.selectAll('g.hovertext');
+            function assertHoverText(xLabel, yLabel, zLabel) {
+                var node = d3.selectAll('g.hovertext');
                 expect(node.size()).toEqual(1, 'one hover text group');
 
-                node = d3.selectAll('g.hovertext').selectAll('tspan')[0];
-                expect(node[0].innerHTML).toEqual('x: 140.72', 'x val on hover');
-                expect(node[1].innerHTML).toEqual('y: −96.97', 'y val on hover');
-                expect(node[2].innerHTML).toEqual('z: −96.97', 'z val on hover');
+                var tspan = d3.selectAll('g.hovertext').selectAll('tspan')[0];
+                expect(tspan[0].innerHTML).toEqual(xLabel, 'x val on hover');
+                expect(tspan[1].innerHTML).toEqual(yLabel, 'y val on hover');
+                expect(tspan[2].innerHTML).toEqual(zLabel, 'z val on hover');
+            }
+
+            it('should have', function(done) {
```

Comment:
This isn't new with this PR, but this labeling is a little weird - it looks like you're including the message ('x val on hover' etc) as part of the test description, but doing that doesn't make it read out the right way either in the code or when the test fails.

The way I understand it, these tests should be described as:
```js
describe('Noun', function() {
    it('makes a sentence when you include the "it"', function() {
        expect(true).toBe(false, 'more info if useful pinpoint the problem');
    });
});
```

Which, when it fails, will make a message that starts with a sentence:

```
Chrome 56.0.2924 (Mac OS X 10.12.3) Noun makes a sentence when you include the "it" FAILED
        Expected true to be false, more info if useful pinpoint the problem.
            <traceback>
```

So in this case I'd have expected it to be something like:

```js
describe('gl plot interactions', function() {
    it('makes the right hover text and point data', function() {
        // I'm not sure if the messages add much, I typically only include them
        // when they include some variables either to determine which loop entry
        // we failed on or some input to the calculation that led to the failure
        // but having them doesn't detract from the test usability so you're
        // welcome to keep them if you find them useful
        expect(ptData.x).toBe('140.72');
    });
});
```